### DISCUSSION TITLE
Larger Imgs when in the single rom view mode

### DISCUSSION THREAD

**Discussion by flagone** [OnionUI/Onion#540] [Ideas]
The rom list either displays multiple roms and a small screenshot, or a single rom and a larger screenshot.

However, when viewing a single rom, the larger screenshot is still tiny.  

In my view, it would be great if the larger screenshot could fill most of the display with the rom name listed underneath.

**Comment by Olywa** [comment]
You mean larger images on the 'info' screen I think (dpad right from game list screen), A few have asked for this but no one has figured out how to do it yet.

**Comment by flagone** [comment]
Yes, it was larger images on the info screen.  It's good to know it is on the radar if someone works out how to do it.
